code/style.css

```diff
@@ -1,4 +1,194 @@
+            justify-content: space-between;
+            align-items: center;
+            height: 100px;
+            padding: 0 20px;
+        }
+        
+        .hamburger {
+            display: none;
+            width: 30px;
+            height: 30px;
+            flex-direction: column;
+            justify-content: space-between;
+            margin-right: 24px;
+        }
+        
+        .hamburger div {
+            width: 30px;
+            height: 5px;
+            background: #fff;
+        }
+        
+        nav {
+            margin-right: 50px;
+        }
+        
+        nav a {
+            color: #fff;
+            text-decoration: none;
+            text-transform: uppercase;
+            margin-left: 20px;
+            font-size: 20px;
+            color: #1761a0
+        }
+        /*--------------main--------------------*/
+        
+        .wrapper {
+            position: relative;
+            min-height: 100%;
+        }
+        
+        .info {
+            width: 100%;
+            height: auto;
+            display: flex;
+        }
+        
+        .info img {
+            max-height: 500px;
+            width: 70%;
+            /* align-self: center; */
+        }
+        
+        .info .text {
+            align-self: center;
+            margin-left: auto;
+            margin-right: auto;
+            text-align: center;
+            width: 30%;
+        }
+        
+        .btn {
+            background: #f0a418;
+            padding: 10px;
+            color: #fff;
+            margin-left: auto;
+            border-radius: 15px;
+            color: #1761a0;
+        }
+        
+        .destinations {
+            width: 100%;
+            height: auto;
+            margin-top: 40px;
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: center;
+        }
+        
+        .destination-box {
+            width: 20%;
+            height: auto;
+            background: #ecf2f7;
+            margin: 20px;
+            padding: 10px;
+            box-sizing: border-box;
+            box-shadow: 0 10px 10px #999;
+            transition: box-shadow 0.3s ease
+        }
+        
+        .destination-box:hover {
+            box-shadow: 5px 10px 10px #999;
+        }
+        
+        .destination-box img {
+            height: 200px;
+            width: 100%;
+        }
+        
+        .destinations h2 {
+            text-align: center;
+        }
+        
+        .staycation {
+            height: auto;
+            display: grid;
+            grid-template-columns: repeat(6, 1fr);
+            margin-top: 100px;
+            margin-bottom: 100px;
+            background: #ecf2f7;
+            grid-gap: 5px;
+        }
+        
+        .staycation .grid2 {
+            padding: 50px;
+            text-align: center;
+        }
+        
+        .staycation img {
+            width: 100%;
+            height: 200px;
+        }
```

Comment:
The fixed height makes the images a bit stretched in mobile view. If you have a width 100% and don't define the height, the images will adjust and not looked stretched. 

If you where to do two changes for the mobile styling:
1) Remove the fixed height of the images
2) Change the width of the destinations-box class to be something like 90% 
The nice polariod effect you created would pop even more, just a tip for the future 🤗